stdlib/_curses.pyi

```diff
@@ -61,7 +61,8 @@ if sys.platform != "win32":
     A_DIM: int
     A_HORIZONTAL: int
     A_INVIS: int
-    A_ITALIC: int
+    if sys.platform != "darwin":
+        A_ITALIC: int
```

Comment:
Docs here: https://docs.python.org/3/library/curses.html#curses.A_ITALIC

Immediately above this table, the docs state:

> Some constants are available to specify character cell attributes. The exact constants available are system dependent.

Reply:
I tried and it indeed exists on Linux but not macOS